src/tools/projects.md

```diff
@@ -154,6 +154,3 @@ starting a new project using anything in this section.
 
 | Name and repo                  | Description                             |
 |--------------------------------|-----------------------------------------|
-| [`Fx-Data-Planning`][planning] | Quarterly goals and internal documentation
```

Comment:
This is now an empty table, which I guess isn't great, but oh well. We should just move this whole page somewhere more appropriate.